taqueria-plugin-taquito/originate.ts

```diff
@@ -165,20 +165,64 @@ const createBatch = async (parsedArgs: Opts, tezos: TezosToolkit, destination: s
 		if (error.message) {
 			const msg = error.message;
 			if (/ENOTFOUND/.test(msg)) {
-				sendErr(msg + ' - The RPC URL may be invalid. Check your ./taq/config.json.');
+				sendErr('The RPC URL may be invalid. Check your ./taq/config.json.\n');
```

Comment:
```suggestion
				sendErr('The RPC URL may be invalid. Check your ./.taq/config.json.\n');
```
Also, adding the JSON path might help the user